<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1450</width>
    <height>860</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="loginBg">
     <property name="styleSheet">
      <string notr="true">#loginBg
{
	background-color:rgb(255, 255, 255);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="head" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>68</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>68</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>68</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>68</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{

	border:none;
	

}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>36</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>36</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>1297</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="minBtn">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">#minBtn
{
	border-image:url(&quot;:/image/minilize.png&quot;)


}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="quitBtn">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">#quitBtn{
	border-image: url(&quot;:/image/closeBtn.png&quot;);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="body" native="true">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>421</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>421</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="content" native="true">
           <property name="styleSheet">
            <string notr="true">#content{
	font-family: &quot;微软雅黑&quot;;
	font-size: 22px;
 	color: black;
}</string>
           </property>
           <widget class="QLabel" name="logLabel">
            <property name="geometry">
             <rect>
              <x>160</x>
              <y>60</y>
              <width>280</width>
              <height>78</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">#logLabel {
    color: black;                     
    font-weight: 700;                  
    font-family: &quot;微软雅黑&quot;;
    font-size: 48px;              
    qproperty-alignment: 'AlignCenter';
    margin: 10px 0;               
    letter-spacing: 1px;           
}</string>
            </property>
            <property name="text">
             <string>VisNova</string>
            </property>
           </widget>
           <widget class="QPushButton" name="pdBtn">
            <property name="geometry">
             <rect>
              <x>3</x>
              <y>208</y>
              <width>300</width>
              <height>63</height>
             </rect>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">#pdBtn
{
	color: #222222;
	border:none;
	border-bottom: 2px solid #B5ECFF;
	font-size: 22px;

}</string>
            </property>
            <property name="text">
             <string>密码登录</string>
            </property>
           </widget>
           <widget class="QPushButton" name="msgBtn">
            <property name="geometry">
             <rect>
              <x>303</x>
              <y>208</y>
              <width>300</width>
              <height>63</height>
             </rect>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">#msgBtn
{
	color: #3ECEFF;
	font-weight: bold;
	font-size: 22px;
	border:none;
	border-bottom: 6px solid #3ECEFF;
}</string>
            </property>
            <property name="text">
             <string>短信登录</string>
            </property>
           </widget>
           <widget class="QWidget" name="pdContain" native="true">
            <property name="geometry">
             <rect>
              <x>3</x>
              <y>270</y>
              <width>600</width>
              <height>374</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel{
	font-size: 18px;	
	font-weight: bold;
	color: black;
}

QFrame{
	background-color: #F5F4F5;
	border-radius: 36px
}

QLineEdit{
	font-size: 18px;
	border:none;	
	background-color: #F5F4F5;
	color: black;
}</string>
            </property>
            <widget class="QFrame" name="accountBox">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>35</y>
               <width>600</width>
               <height>72</height>
              </rect>
             </property>
             <property name="frameShape">
              <enum>QFrame::Shape::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Shadow::Raised</enum>
             </property>
             <widget class="QLabel" name="atLabel">
              <property name="geometry">
               <rect>
                <x>40</x>
                <y>24</y>
                <width>36</width>
                <height>24</height>
               </rect>
              </property>
              <property name="text">
               <string>账号</string>
              </property>
             </widget>
             <widget class="QLineEdit" name="atEdit">
              <property name="geometry">
               <rect>
                <x>124</x>
                <y>20</y>
                <width>400</width>
                <height>25</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">color: black;</string>
              </property>
              <property name="echoMode">
               <enum>QLineEdit::EchoMode::Normal</enum>
              </property>
              <property name="placeholderText">
               <string>请输入手机号或用户昵称...</string>
              </property>
             </widget>
            </widget>
            <widget class="QFrame" name="passwordBox">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>143</y>
               <width>600</width>
               <height>72</height>
              </rect>
             </property>
             <property name="frameShape">
              <enum>QFrame::Shape::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Shadow::Raised</enum>
             </property>
             <widget class="QLabel" name="pdLabel">
              <property name="geometry">
               <rect>
                <x>40</x>
                <y>24</y>
                <width>36</width>
                <height>24</height>
               </rect>
              </property>
              <property name="text">
               <string>密码</string>
              </property>
             </widget>
             <widget class="QLineEdit" name="pdEdit">
              <property name="geometry">
               <rect>
                <x>124</x>
                <y>20</y>
                <width>400</width>
                <height>25</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">color: black;</string>
              </property>
              <property name="echoMode">
               <enum>QLineEdit::EchoMode::Password</enum>
              </property>
              <property name="placeholderText">
               <string>请输入密码...</string>
              </property>
             </widget>
            </widget>
            <widget class="QPushButton" name="registerNowForpd">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>299</y>
               <width>290</width>
               <height>72</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">#registerNowForpd{
	color: #222222;
	border-radius: 36px;
	border: 1px solid #C2C9D9;
	font-size: 22px;
}</string>
             </property>
             <property name="text">
              <string>立即注册</string>
             </property>
            </widget>
            <widget class="QPushButton" name="loginNowForpd">
             <property name="geometry">
              <rect>
               <x>310</x>
               <y>299</y>
               <width>290</width>
               <height>72</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">#loginNowForpd{
	color: #FFFFFF;
	background-color: #3ECEFE;
	border-radius: 36px;
	font-size: 22px;
}</string>
             </property>
             <property name="text">
              <string>登录</string>
             </property>
            </widget>
           </widget>
           <widget class="QWidget" name="msgContain" native="true">
            <property name="geometry">
             <rect>
              <x>3</x>
              <y>270</y>
              <width>600</width>
              <height>374</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel{
	font-size: 18px;
	font-weight: bold;	
	color: black;

}

QLineEdit{
	border: none;
	font-size: 18px;
	color: black;
	background-color: #F5F4F5;
}


QFrame{
	background-color: #F5F4F5;
	border-radius: 36px;
}

QPushButton{
	background-color: #F5F4F5;

}</string>
            </property>
            <widget class="QFrame" name="emailBox">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>35</y>
               <width>600</width>
               <height>72</height>
              </rect>
             </property>
             <property name="frameShape">
              <enum>QFrame::Shape::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Shadow::Raised</enum>
             </property>
             <widget class="QLabel" name="emailLabel">
              <property name="geometry">
               <rect>
                <x>40</x>
                <y>24</y>
                <width>40</width>
                <height>24</height>
               </rect>
              </property>
              <property name="text">
               <string>+86</string>
              </property>
             </widget>
             <widget class="QLineEdit" name="emailEdit">
              <property name="geometry">
               <rect>
                <x>108</x>
                <y>24</y>
                <width>350</width>
                <height>24</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">color: black;</string>
              </property>
              <property name="placeholderText">
               <string>输入手机号...</string>
              </property>
             </widget>
             <widget class="QPushButton" name="authCodeBtn">
              <property name="geometry">
               <rect>
                <x>470</x>
                <y>20</y>
                <width>93</width>
                <height>28</height>
               </rect>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">#authCodeBtn{
	color: #999999;

	font-size: 18px;
	border: none;
}</string>
              </property>
              <property name="text">
               <string>获取验证码</string>
              </property>
             </widget>
            </widget>
            <widget class="QFrame" name="authcodeBox">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>143</y>
               <width>600</width>
               <height>72</height>
              </rect>
             </property>
             <property name="frameShape">
              <enum>QFrame::Shape::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Shadow::Raised</enum>
             </property>
             <widget class="QLineEdit" name="autoCodeEdit">
              <property name="geometry">
               <rect>
                <x>108</x>
                <y>24</y>
                <width>350</width>
                <height>24</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">color: black;</string>
              </property>
              <property name="placeholderText">
               <string>请输入验证码</string>
              </property>
             </widget>
             <widget class="QLabel" name="autoCodeLabel">
              <property name="geometry">
               <rect>
                <x>40</x>
                <y>24</y>
                <width>36</width>
                <height>24</height>
               </rect>
              </property>
              <property name="text">
               <string>密码</string>
              </property>
             </widget>
            </widget>
            <widget class="QPushButton" name="loginOrRegister">
             <property name="geometry">
              <rect>
               <x>2</x>
               <y>298</y>
               <width>596</width>
               <height>72</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">#loginOrRegister{
	color: #FFFFFF;
		background-color: #3ECEFE;
	border-radius: 36px;
	font-size :22px;

}</string>
             </property>
             <property name="text">
              <string>登录/注册</string>
             </property>
            </widget>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
